test/npm.js

```diff
@@ -1,24 +1,32 @@
 /* eslint-disable promise/prefer-await-to-callbacks */
 
+
+const run = function run (args, map, cb) {
+  let res = map;
+  for (const arg of args) {
+    res = res[arg];
```

Comment:
what is the purpose of this reassigning?

Reply:
To crawl through the arguments and the data structure npm provides.

Reply:
It would be useful to add some comment there